### DISCUSSION TITLE
Chained merging

### DISCUSSION THREAD

**Discussion by miasik** [wkpark/sd-webui-model-mixer#79] [Ideas]
I'd like to see something like the next.
An interface to create a set of chained steps:
Result1=Model1+Model2(mbw array1)
Result2=Result1+Model3(mbw array2)
Result3=Result2+Model4(mbw array3)
Result4=Result3+Model5(mbw array3)
The steps are being executed one-by-one.

Merge board can do it, but without mbw https://github.com/bbc-mc/sdweb-merge-board

**Comment by wkpark** [comment]
> I'd like to see something like the next. An interface to create a set of chained steps:
> - Result1=Model1+Model2(mbw array1)
> - Result2=Result1+Model3(mbw array2)
> - Result3=Result2+Model4(mbw array3)
> - Result4=Result3+Model5(mbw array3)
> The steps are being executed one-by-one.
> 
> Merge board can do it, but without mbw https://github.com/bbc-mc/sdweb-merge-board

Currently, Model-Mixer does exactly what you said in one step. (any **sequential** merge possible)

**Reply by miasik** [reply]
Oups! Looks like I should check more carefully :-D

**Reply by miasik** [reply]
I've just noticed that I have Mixer installed and see it in Settings, but don't have a tab for it :-(

**Comment by wkpark** [comment]
> I've just noticed that I have Mixer installed and see it in Settings, but don't have a tab for it :-(

Model mixer does not have a Tab, It has txt2img and img2img Accordions

**Comment by miasik** [comment]
All done here, thanks!
